packages/react/src/components/Dialog/Dialog.test.tsx

```diff
@@ -233,6 +234,65 @@ test('should set alignment on dialog footer', () => {
   expect(footer).toHaveClass('text--align-right');
 });
 
+test('it should call "onClose" when escape is pressed', async () => {
```

Comment:
This is great! This is coverage that we probably should've had before but didn't.